src/app/webserver/websites/binding-list.component.ts

```diff
@@ -283,7 +280,7 @@ export class BindingItem implements OnInit, OnChanges {
             this.model.protocol = "https";
 
             if (!this.model.certificate) {
-                this._certSelect.changes.first().subscribe(c => {
+                this._certSelect.changes.pipe(first()).subscribe(c => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.